examples/ecs/send_and_receive_events.rs

```diff
@@ -0,0 +1,174 @@
+//! From time-to-time, you may find that you want to both send and receive and event of the same type in a single system.
```

Comment:
This shouldn't actually have hyphens I think (also typo)
```suggestion
//! From time to time, you may find that you want to both send and receive an event of the same type in a single system.
```